apps/gp2-server/scripts/import-users.ts

```diff
@@ -82,6 +82,7 @@ const app = async () => {
         firstName: data[0]!,
         lastName: data[1]!,
         country: data[4]!,
+        stateOrProvince: 'Unknown',
```

Comment:
I'm not sure if this script will ever be used again, but it was giving a TS error without this line. So I've added it. Maybe we could delete this in a following PR?